packages/core/misc-widgets/src/components/github-star/README.md

```diff
@@ -0,0 +1,24 @@
+# Github Star
+
+A Github Star component for displaying number of stars received by the repo on Github
+
+- [Features](#features)
+- [Usage](#usage)
+  - [Props](#props)
+
+## Features
+
+- Render number of stars received by the repo on Github
+- Customize the repo url that opens in a new tab upon clicking this component
+
+## Usage
+
+### Props
+
+#### `url`
+
+- type: `String`
+- required: `false`
```

Comment:
I believe the `url` prop should be documented as required since the component isn't functional without it.

Reply:
👍 I was in two minds, if we should display the button regardless or not.

Reply:
Updated to required